apps/extension/src/routes/popup/settings/settings-rpc.tsx

```diff
@@ -24,7 +23,8 @@ export const SettingsRPC = () => {
       try {
         await querier.chainParams();
         await setGRPCEndpoint(rpc);
-        await internalSwClient.clearCache();
+        const cleared: boolean = await chrome.runtime.sendMessage('PENUMBRA_CLEAR_CACHE');
```

Comment:
Manually annotating the types here is also not so great. The consumers of these should get the typing automatically. If not, it will not stay in sync if changed.

Reply:
using a 'blockCacheControl' interface now